exceptions.txt

```diff
@@ -419,3 +419,6 @@
 294973: 'Home Fires UK',
 295438: 'Shark UK',
 148581: 'Strike Back Legacy',
+281714: 'Ballers', 'Ballers (2015)', 'Ballers (2014)',
+281708: 'Proof', 'Proof (US)',
+294417: 'Scream The TV Series',
```

Comment:
I looked at the list and I don't see a scene exception for Scream... I'm super new to this, so I'm sure I'm missing it. 

btw, trying to figure this out on my own. I'd love to contribute to the scene exceptions list and/or other areas due to the ridiculous amount of TV series I download/watch ... I'm just a little new to github and SR. 

(ps - just sent a donation. But, it is coming from a different email. Thanks again)